src/sql/src/session/vars.rs

```diff
@@ -664,6 +667,15 @@ const DISK_CLUSTER_REPLICAS_DEFAULT: ServerVar<bool> = ServerVar {
     internal: true,
 };
 
+const UNSAFE_NEW_TRANSACTION_WALL_TIME: ServerVar<Option<CheckedTimestamp<DateTime<Utc>>>> = ServerVar {
+    name: UncasedStr::new("unsafe_new_transaction_wall_time"),
+    value: &None,
+    description:
+        "Sets the wall time for all new explicit or implicit transactions to control the value of `now()`. \
+        If not set, uses the system's clock.",
+    internal: false,
```

Comment:
It needs to be false because we want non-system user access. Added a comment describing this.

Reply:
You could also execute the test as the mz_system role. Something like
```
simple conn=mz_system,user=mz_system
ALTER SYSTEM SET enable_rbac_checks TO true;
SELECT * FROM pg_timezone_names
----
...
COMPLETE _
```

Though I think the unsafe probably works fine too.

Reply:
That doesn't work because this is a session var, not a server var. It can't be a server var because we start transactions in the adapter, not coordinator, and we don't have access to the system vars there (without introducing a new coord message which seems utterly unnecessary for this).

Reply:
I'm pretty sure that the `conn=mz_system` part creates a long lived session with the label `mz_system`, that will remember changes to session variables. Any other statement executed with the same `conn` label with be executed in the same session. Still, I think it's fine as is, just an FYI.

Reply:
Indeed. I wanted to be able to use non-`simple` slt directives. Agree it's fine as is.